app/controllers/sessions_controller.rb

```diff
@@ -4,24 +4,24 @@ def new
   end
 
   def create
-    if logging_user&.authenticate(params[:session][:password])
+    if login_sucess
       log_in logging_user
       flash[:success] = t('.success_login')
       redirect_to logging_user
     else
-      flash[:warning] = t('.wrong_password')
+      flash.now[:warning] = t('.wrong_password')
       render 'new'
     end
   end
 
   def destroy
-    log_out
+    session.delete(:user_id)
+    @current_user = nil
     redirect_to root_url
   end
 
-  def log_out
-    session.delete(:user_id)
-    @current_user = nil
+  def login_sucess
```

Comment:
`valid_account?`

Reply:
already change the name of the def to authenticate_sucess